spec/index.bs

```diff
@@ -2641,12 +2641,12 @@ Using external textures costs several binding points: see [=binding cost=].
     The configuration used may not be stable across time, systems, user agents, media sources,
     or frames within a single video source.
     In order to account for many possible representations,
-    the [=binding cost=] conservatively requires:
+    the [=binding cost=] conservatively requires, for *each* external texture:
 
-    - for pipeline layouts that have *any* external textures, one sampler binding
-        for sampling 3D LUTs.
-    - for *each* external texture, one sampled texture binding for each plane,
-        one sampled texture binding for a 3D LUT, and one uniform buffer binding for metadata.
+    - three sampled texture bindings (for up to 3 planes),
```

Comment:
I wonder if it would be easier to just give it the whole bind group

Reply:
It doesn't help for the limit because they are for the whole pipeline layout.

Reply:
Yes, but I wonder how the discussion goes about bind group re-creation. If we force users to re-create bind groups, we might as well just accept `GPUExternalTexture` *instead* of a bind group.

Reply:
re-create bind groups when, for different external textures? I was imagining we would always use an exact number of bindings in both the bind group layout and the bind group regardless of the underlying representation, and bind dummy objects in any unused slots.

Reply:
re-create every frame, according to https://github.com/gpuweb/gpuweb/pull/1666#discussion_r626204783

Reply:
I don't think there's a good reason to use up, by design, an entire bind group slot for one external texture. I think there are reasonable applications that would need to bind two or more external textures (e.g. compositing a foreground canvas over a background webcam feed) but not want to use up two of their four bind groups.